packages/design-system/src/scalars/components/radio-group/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useId } from "react";
+import * as RadioGroupPrimitive from "@radix-ui/react-radio-group";
+import { FormLabel } from "@/scalars/components/form-label";
+import { cn } from "@/scalars/lib/utils";
+
+export interface RadioGroupProps
+  extends React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Root> {
+  "aria-label"?: string;
+  children: React.ReactElement;
+  className?: string;
+  defaultValue?: string;
+  description?: string;
+  hasError?: boolean;
```

Comment:
We should add an `error` props (it's a requirement), from that prop you can define if there's errors or not

Reply:
Done